clock/pom.xml

```diff
@@ -51,6 +51,12 @@
             <groupId>org.redisson</groupId>
             <artifactId>redisson</artifactId>
             <scope>test</scope>
+            <exclusions>
+                <exclusion>
```

Comment:
Why do we need to exclude this? In general, we try to avoid exclusions unless absolutely necessary, as they are hard to manage long term.